CMakeLists.txt

```
@@ -18,8 +18,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE_CXX ${CMAKE_CURRENT_SOURCE_DIR}/cmake/cxx_fla
 
 project(polserver CXX C)
 
-set(POL_VERSION_STR "100")
-set(POL_VERSION 100)
+set(POL_VERSION_STR "100.0.0")
```

We could also remove POL_VERSION_STR completely. But this would work for now:
```suggestion
set(POL_VERSION_STR "${polserver_VERSION}")
```
Cmake also defines `polserver_VERSION_MAJOR`, `polserver_VERSION_MINOR` etc for us.

---

cool!